documentation/troubleshooting/modules/ipi-install-troubleshooting-misc-issues.adoc

```diff
@@ -152,3 +152,14 @@ NOT* approve them, therewise you will face even more issues.
 # If you see Subject Name: localhost.localdomain remove the csr
 [kni@provisioner ~]$ oc delete csr <wrong_csr>
 ----
+
+== Deployment failures due to VIP's being used by rogue nodes 
+
+When re-deploying the cluster with same name please make sure to turn off all the nodes including the once from previous install. 
```

Comment:
During the installation process, it is possible to encounter a VRRP conflict if a previously used {product-title} node that was once part of a cluster deployment using a specific cluster name is still running but not part of current {product-tile} cluster deployment using that same cluster name. 

For example, imagine a scenario were a deployment was done using the cluster name `openshift` deploying 3 masters nodes and 3 workers nodes. Once this install was complete, a future install was started using the same cluster name `openshift` but this redeployment only installed 3 master nodes, leaving the 3 worker nodes from a previous deployment in an `ON` state. This would cause a VRID conflict and VRRP errors 

.Solution
This scenario has two solutions.
. Ensure that the `cluster name` for every deployment is unique ensuring no conflict.
. Turn off all the rogue nodes which are not part of the cluster deployment that are using the same `cluster name`. Otherwise, the authentication pod of your {product-title} cluster may never start successfully.

Reply:
thanks @rlopez133 , i have incorporated the changes.